src/general_manager/interface/models.py

```
@@ -0,0 +1,80 @@
+from __future__ import annotations
+from typing import Type, ClassVar, Any, Callable, TYPE_CHECKING, TypeVar, Generic
```

Flake8 (7.2.0)

[error] 'typing.Generic' imported but unused

(F401)